Specs/Core/CesiumTerrainProviderSpec.js

```diff
@@ -1168,14 +1168,15 @@ describe("Core/CesiumTerrainProvider", function () {
       });
     });
 
-    xit("Uses query parameter extensions for ion resource", function () {
+    it("Uses query parameter extensions for ion resource", function () {
       const terrainProvider = new CesiumTerrainProvider({
-        url: IonResource.fromAssetId(1),
+        url: "/Specs/Mocks/CesiumTerrainProvider", // Mock payload from Ion
         requestVertexNormals: true,
         requestWaterMask: true,
+        ready: true,
       });
 
-      return pollToPromise(function () {
+      pollToPromise(function () {
```

Comment:
You need to return the result of this function. By returning a promise, the tests will wait until the promise has been fulfilled and all expectations in this promise chain have been tested. Otherwise, the test will finish early and can interfere with subsequent tests.